plugins/inventory/linode.py

```diff
@@ -78,6 +78,7 @@
   webservers: "'web' in (tags|list)"
   mailservers: "'mail' in (tags|list)"
 compose:
+  ansible_ssh_host: ipv4[0]
```

Comment:
Maybe add a comment that explains what this does, for example:
```suggestion
  # By default, Ansible tries to connect to the label of the instance.
  # Since that might not be a valid name to connect to, you can
  # replace it with the first IPv4 address of the linode as follows:
  ansible_ssh_host: ipv4[0]
```

Reply:
Done!